palak/src/components/BackGround.jsx

```diff
@@ -0,0 +1,73 @@
+import { Link } from "react-router-dom";
+import Login from "../pages/Login";
+import ForgotPsswrd from "../pages/ForgotPsswrd";
+
+const BackGround = ({
+  type,
+  handleTypeChange,
+  formData,
+  error,
+  handleDataChange,
+  handleDataSubmit,
+}) => {
+  return (
+    <div className="bg-cover h-screen bg-[url('src/assets/bg.jpg')]">
+      <Link to="#" className="grid">
+        <div className="flex items-start lg:mb-8 justify-center lg:mt-8 lg:mx-20 md:justify-start sm:my-2 md:my-2">
```

Comment:
Try to avoid adding breakpoints to sm, md and lg all three together. Check to fix this.